src/Orleans.PlatformServices/Orleans.PlatformServices.csproj

```diff
@@ -31,4 +31,8 @@
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.5'">
     <PackageReference Include="System.Runtime.Loader" Version="$(SystemRuntimeVersion)" />
   </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Orleans.Core.Abstractions\Orleans.Core.Abstractions.csproj" />
```

Comment:
Not really sure about this one. PlatformServices should probably remain agnostic of anything related to Orleans. So I would remove it from here unless it's absolutely necessary in the future